docs/release-notes/NuGet-6.11.md

```diff
@@ -0,0 +1,101 @@
+---
+title: NuGet 6.11 Release Notes
+description: Release notes for NuGet 6.11 including new features, bug fixes, and DCRs.
+author: martinrrm
+ms.date: 8/1/2024
+ms.topic: conceptual
+---
+# NuGet 6.11 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.11**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.11](https://visualstudio.microsoft.com/downloads/) | [8.0.4xx](https://dotnet.microsoft.com/download/dotnet/8.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
+
+## Summary: What's New in 6.11
+
+* Suppress NuGetAudit warnings for specific advisories for PackageReference projects - [#13679](https://github.com/NuGet/Home/issues/13679)
+
+* Add `--allow-insecure-connections` option to dotnet SDK - [#13396](https://github.com/NuGet/Home/issues/13396)
+
+* Swap authors for owners in Visual Studio Package Manager UI - [#12501](https://github.com/NuGet/Home/issues/12501)
+
+* [Feature]: add dotnet nuget why to dotnet CLI - [#11943](https://github.com/NuGet/Home/issues/11943)
+
+* NuGet cannot restore from HTTPS sources that have SSL certificate problems - [#4387](https://github.com/NuGet/Home/issues/4387)
+
+### Breaking changes
+
+* `MSBuildRestoreUtility.GetRestoreAuditProperties` needs a breaking change to read `NuGetAuditSuppress` items - [#13313](https://github.com/NuGet/Home/issues/13313)
+
+* NuGetAudit should check transitive packages by default when the .NET 9 SDK is installed - [#13293](https://github.com/NuGet/Home/issues/13293)
```

Comment:
This one is tricky. It technically matches 9.0 and VS 17.12, but it's something that could be helpful for people to know.n

Reply:
yeah, I wasn't sure about commenting on this one either.

I don't think there are any scenarios where this really matters to customers. Even if the customer is using nuget.exe, I don't think it matters if they're using nuget.exe 6.11 or 6.10, in both cases I'd expect the project to load the SDK's nuget.targets, not nuget.exe's targets, and therefore it's still the version of the SDK that matters, not the version of NuGet.

But the change to nuget.targets, that adds the version check for the .NET 9 SDK, is added to nuget 6.11. 🤷

Reply:
> Even if the customer is using nuget.exe, I don't think it matters if they're using nuget.exe 6.11 or 6.10, in both cases I'd expect the project to load the SDK's nuget.targets, not nuget.exe's targets, and therefore it's still the version of the SDK that matters, not the version of NuGet

I actually think it's opposite. In fact, I'm pretty sure it is: https://github.com/NuGet/NuGet.Client/blob/9fbabf9a119599b62824f676407c5611c0542781/src/NuGet.Clients/NuGet.CommandLine/MsBuildUtility.cs#L263.

You *still* need .NET 9 of the SDK. 

It almost feels like this particular issue should be referenced in both 6.11 and 6.12 since 6.12 matches .NET 9.

Reply:
Since it's in the breaking change section I think leave it is better

Reply:
Should we call this a feature instead of a breaking change? 

@zivkan Our breaking change section was meant to be for APIs. This is more of an addition/roll out.

Reply:
Sorry, I forgot to reply to this earlier. In case it's still useful, we can continue the conversation

> Our breaking change section was meant to be for APIs

Where was this communicated? If this is not obvious to customers reading the release notes, then I disagree that we should keep it for API breaking changes only. If a customer is experiencing problems with a new release, it's faster for them to check the release notes and look at a short list of breaking changes, compared to (what's hopefully) a longer list of features.

Reply:
I feel like the new feature section gets more prominence the breaking changes section by virtue of headings. 
I added the original breaking changes section and that was pretty much my interpretation.

Another thing is that transitive vulnerabilities aren't a traditional breaking chnage, they're a new feature. I'd rather have the precedent where we can make calls like this. 

Honestly, having a flagship feature in the breaking changes section doesn't do it the service it deserves it.